__tests__/actions/URLEncode.spec.ts

```diff
@@ -32,4 +43,20 @@ describe('URLEncode function', () => {
     expect(actual).toEqual(expected);
   });
 
+  it('inverts a URLEncode action', () => {
+    if (URLEncode.invert) {
+      const action = {
+        WFWorkflowActionIdentifier: 'is.workflow.actions.urlencode' as WFWorkflowActionIdentifier,
+        WFWorkflowActionParameters: {
+          WFEncodeMode: '' as WFWorkflowActionParameters['WFEncodeMode'],
```

Comment:
Can this be please be changed to:

```ts
WFEncodeMode: '' as WFEncodeMode,
```
You'll then also need to make sure `WFEncodeMode` is imported at the top, and you can then remove the `WFWorkflowActionParameters` import again.